src/app/client/src/app/modules/learn/services/course-consumption/course-consumption.service.ts

```diff
@@ -23,7 +23,8 @@ export class CourseConsumptionService {
   showJoinCourseModal = new EventEmitter<any>();
   enableCourseEntrollment = new EventEmitter();
   coursePagePreviousUrl: any;
-  userCreatedAnyBatch = new EventEmitter();
+  public _isMentorOfAnyBatch = false;
```

Comment:
You should not add this variable here. It should be similar to canCreateBatch() method.